src/Card/Card.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import { cardAlignOptions } from "./Card.levers";
+import { FlexBox } from "../FlexBox/FlexBox";
+
+/**
+ * Card Component
+ *
+ * HTML card element
+ */
+export const Card = ({
+  className,
+  children,
+  image,
+  align,
+  isMinimal,
+  ...props
+}) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+
+  let wrapperClasses =
+    "su-bg-white su-text-black su-border su-border-solid su-border-transparent-black su-shadow";
```

Comment:
Quick tip: su-border-transparent-black is only available in the SAA site (we added this color in the the SAA homesite TW color theme which extends the Decanter v7 theme colors). You could possibly sub it using su-border-black-10 (which is 10% black in the Identity Guide) or even su-foggy-light - see which one Kerri signs off on 😜